### PR TITLE
Proposal for improving overall formats

### PR DESCRIPTION
**Pull request by temeddix**
## Changes

Hi, this is a proposal rather than an actual pull request. There are 2 suggestions in this PR:

- Format all `toml` `yaml` `yml` `md` files via Prettier formatter. There's not any workflow to check the format of those filetypes yet. This PR only includes formatted files. However, simply doing this will improve consistency.
- Reorganize README.md file a little. I am relatively new to flutter_rust_bridge, and when I first saw the README it felt a bit clunky to me😅. To be specific, project description was on the title, other project `flutter_smooth` was at the middle, badge was suddenly being non-rounded, the main link to user guide was not clear and '<->' looked like a code rather than a description. I think changes in README from this PR can improve newcomer developer experience.

I also wanted to mention the contributors table having inconsistent column width. But since that's created by the bot and GitHub doesn't allow styling of HTML, I guess that's not on our side to fix.

This is just a humble proposal, so I will delete it if it's not acceptable :)

## Checklist

- [x] An issue to be fixed by this PR is listed above.
- [x] New tests are added to ensure new features are working. End-to-end tests are usually in the `./frb_example/pure_dart` example, more specifically, `rust/src/api.rs` and `dart/lib/main.dart`.
- [x] The code generator is run and the code is formatted (via `just precommit`).
- [x] If this PR adds/changes features, documentations (in the `./book` folder) are updated.
- [x] CI is passing.

## Remark for PR creator

- Justfile is a task runner for the command line interface that allows you to run shell commands from a file named `justfile` in your project directory, You can use Justfile after [installing it](https://github.com/casey/just). Note that commands written in `justfile` of this repository are expected to be run in `bash`, not `cmd` or `powershell`. Running `just ...` commands in `cmd` or `powershell` will produce erorrs as the syntax is not compatible. On Windows, you can use `git bash` if you have Git installed.
- If fzyzcjy does not reply for a few days, maybe he just did not see it, so please ping him.

### CODE REVIEW

File: book/src/build_wasm.md [hunk 1]
```diff
@@ -64,11 +66,12 @@ void main() async {
...
+    support scripts which have not been marked as `crossorigin="anonymous"` and
+    therefore cannot be loaded. For local testing, you can specify `credentialless`
+    instead.
 
 [`wasm_bindgen`]: https://rustwasm.github.io/docs/wasm-bindgen
-[`Cross-Origin-Resource-Policy`]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Cross-Origin_Resource_Policy_(CORP)
-[`Cross-Origin-Embedder-Policy`]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Embedder-Policy
\ No newline at end of file
+[`cross-origin-resource-policy`]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Cross-Origin_Resource_Policy_(CORP)
+[`cross-origin-embedder-policy`]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Embedder-Policy
```
Comment (temeddix) [inline book/src/build_wasm.md:77]: This doesn't break the link:
https://github.com/temeddix/flutter_rust_bridge/blob/improve-formats/book/src/build_wasm.md

**Comment by fzyzcjy** [comment]
Good formatting! Just some nits:

> Reorganize README.md file a little

The non-rounded badge looks like a problem and should be fixed. For other things in the README, maybe we can keep the original.

**Comment by fzyzcjy** [comment]
In addition, maybe we should merge the PR after there are some `justfile` which automates the formatting + there are CI checking this. Otherwise, even my *own* markdown editor will have different format than this, so in the next few commits all these formats will be broken again :/

**Comment by fzyzcjy** [comment]
👀

**Comment by temeddix** [comment]
> 👀

It was just a proposal made with auto formatting :) I'll be updating with proper just file commands next time if I can

**Comment by fzyzcjy** [comment]
Sure, take your time :)

**Pull request closed without merging**